jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/composite/MobileScreen.java

```diff
@@ -92,6 +97,21 @@ public static void scrollLeft(int offset) {
         scrollHorizontally(offset);
     }
 
+
+    private static int scrollDivider = 3;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6346206)